sql/moz-fx-data-shared-prod/mozilla_vpn_derived/exchange_rates_v1/query.py

```diff
@@ -73,7 +73,7 @@ class Quote:
                 quote_currency=currency_pair["quoteCurrency"],
                 base_currency=currency_pair["baseCurrency"],
                 price_type=currency_pair["type"],
-                price=price,
+                price=round(Decimal(price), 9),
```

Comment:
Or we could change the `price` column type to be `BIGNUMERIC`, but this approach seemed simplest and 9 decimal places seems plenty accurate.